resources/pedagogy.md

```diff
@@ -17,7 +17,7 @@ If you have questions or need assistance, please reach out to the [Pedagogy Inte
 ## Selected Bibliography
 
 <div class="columns">
-    <div class="column col-12 bibliography">
-        <script src="https://bibbase.org/show?bib=https%3A%2F%2Fapi.zotero.org%2Fgroups%2F4439204%2Fitems%3Fkey%3D8lnTbafaItBb4ZVOjRCEYliR%26format%3Dbibtex%26limit%3D100&jsonp=1&theme=dividers&nocache=1&authorFirst=1"></script>
-    </div>
+ <div class="column col-12 bibliography">
+ <script src="https://bibbase.org/show?bib=https%3A%2F%2Fapi.zotero.org%2Fgroups%2F4439204%2Fitems%3Fkey%3D8lnTbafaItBb4ZVOjRCEYliR%26format%3Dbibtex%26limit%3D100&jsonp=1&theme=dividers&nocache=1&authorFirst=1"></script>
```

Comment:
`script` tag, as a child of `div.bibliography`, needs one more level of indentation